src/mds/MDSMap.cc

```diff
@@ -134,6 +134,8 @@ void MDSMap::dump(Formatter *f) const
 {
   f->dump_int("epoch", epoch);
   f->dump_unsigned("flags", flags);
+  f->dump_unsigned("ever_allowed_features", ever_allowed_features);
```

Comment:
It's awkward to output an integer in JSON output, because to understand it someone will have to come and read the source code to learn what the flags mean.  I'd prefer to see a list of strings here